src/format/scan.rs

```diff
@@ -379,7 +405,7 @@ pub(super) fn timezone_offset_2822(s: &str) -> ParseResult<(&str, Option<i32>)>
             Ok((s, None))
         }
     } else {
-        let (s_, offset) = timezone_offset(s, |s| Ok(s))?;
+        let (s_, offset) = timezone_offset_internal(s, |s| Ok(s), false, false)?;
```

Comment:
Is this making it stricter than it was before?

Reply:
No.

`timezone_offset(.., ..)` calls `timezone_offset_internal(.., .., false)`.

The only effective difference is the newly introduced last function parameter `allow_tz_minus_sign` set to `false` which disallows MINUS-SIGN (U+2212) as a timezone signage. Previous behavior also did not do any handling of MINUS-SIGN (U+2212).